lang/pt_br.php

```diff
@@ -0,0 +1,862 @@
+	'PasswordChangeNote2'     => 'Após alterar sua senha, você será registrado.',
+	'PasswordChangeButton'    => 'Alterar Senha',
+	// - account/changesex
+	'GenderChangeTitle'       => 'Alterar Gênero',
+	'GenderChangeBadChars'    => 'Você não pode trocar de gênero se algum dos seus personagens é um %s',
+	'GenderChanged'           => 'Seu gênero foi alterado com sucesso e $d créditos foram debitados sua conta.',
+	'GenderChangedForFree'    => 'Gênero alterado com sucesso.',
+	'GenderChangeHeading'     => 'Altere Seu Gênero',
+	'GenderChangeCost'        => 'Mudança de Gênero vai custar %s créditos para você.',
+	'GenderChangeBalance'     => 'Você possui atualmente %s créditos.',
+	'GenderChangeNoFunds'     => 'Você não possui créditos suficiente para completar a sua troca de gênero.',
+	'GenderChangeNoCost'      => 'Para você, as mudanças de sexo são livres.',
+	'GenderChangeCharInfo'    => 'Você não pode alterar seu gênero se você tiver um personagem que for um: %s',
+	'GenderChangeSubHeading'  => 'Tenha certeza que você quer realmente alterar!',
+	'GenderChangeFormText'    => 'Você deseja mudar seu gênero para %s?',
+	'GenderChangeConfirm'     => 'Você tem certeza que quer alterar seu gênero?',
+	'GenderChangeButton'      => 'Sim, por favor.',
+	// - account/confirm
+	'AccountConfirmTitle'     => 'Confirmar Conta',
+	'AccountConfirmUnban'     => 'A conta foi confirmada e ativada.',
+	'AccountConfirmMessage'   => 'Sua conta foi confirmada e ativada, agora você pode fazer login.',
+	// - account/confirmemail
+	'EmailConfirmTitle'       => 'Confirmar E-mail',
+	'EmailConfirmFailed'      => 'Tivemos um problema técnico durante sua alteração de email, por favor entre em contato com algum Administrador.',
+	'EmailConfirmChanged'     => 'Seu e-mail foi alterado com sucesso!',
+	// - account/create
+	'AccountCreateTitle'      => 'Criar Uma Conta',
+	'AccountConfirmBan'       => 'Esperando ativação da conta: %s',
+	'AccountCreateEmailSent'  => 'Um e-mail foi enviado contendo as informações de ativação da conta, por favor, cheque seu email e ative a sua conta para poder fazer login.',
+	'AccountCreateFailed'     => 'Sua conta foi criada, mas infelizmente houve uma falha ao lhe enviar o email de confirmação devido a problemas técnicos. Por favor, procure algum GM ou Administrador para resolver o seu problema.',
+	'AccountCreated'          => 'Parabéns! Você foi registrado com sucesso e você já está logado!',
+	'AccountCreateHeading'    => 'Registrar',
+	'AccountCreateTerms'      => 'Termos de Serviço',
+	'AccountCreateInfo'       => 'Por favor, leia os nossos %s antes de criar uma conta, tenha certeza que você entendeu as regras para poder ter uma conta no nosso servidor.',
+	'AccountCreateInfo2'      => 'Clicando em "Criar Minha Conta", você estará concordando com os nossos %s.',
+    'AccountCreateInfo3'      => 'Note: You need to use the email address you provided in order to login.',
```

Comment:
this is caused when you edit files on mac and then in windows, I just added this line